documentation/deployment-devops.md

```diff
@@ -0,0 +1,130 @@
+
+```json
+{
+(((rest of device twin removed for brevity)))
+
+"tags":
+    {"dashboard":true},
+
+(((rest of device twin removed for brevity)))
+}
+```
+
+Save the device twin after the change. Repeat this process for any additional IoT Edge devices that should be targeted with the pipeline.
+
+## Forking the repository
+
+The DevOps pipeline details for the sample are included in [the github repository](https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding).
+
+ Do a GitHub [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) of the repository to your own workspace. After that, continue making changes to the pipeline configuration, for example changing the target conditions.
+
+## Setting up an Azure DevOps organization and project
+
+A Azure DevOps pipeline is always part of a [project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops&tabs=preview-page), which is part of an [organization](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops). Follow the instructions on the given websites, but skip the 'Add a Repository to your Project' part since this is managed on GitHub.
+
+Before adding the pipeline there are two project-level preliminary tasks.
+
+### Install the GitVersion add-in
+
+[GitVersion](https://marketplace.visualstudio.com/items?itemName=gittools.usegitversion) can be used to automatically derive image version tags from a repository. Use the "Get it free" button on the link above to install the add-in into the organization.
+
+### Create a service connection to Azure
+
+A service connection to Azure allows DevOps to push images and create deployments for an Azure subscription.
+
+* In the lower left corner of the pipelin settings, choose "Project Settings"
+* From the left navigation, choose "Service Connections"
+* Click "New Service Connection"
+* Choose "Azure Resource Manager" and hit "next"
+* Choose "Service Principal (automatic)" then "next"
+* Choose an Azure subscription from the dropdown
+  * (For environments where you may not have subscription-level permissions, you may have to also select the specific Resource Group where you deployed your IoT Hub and ACR instance)
+* Add a name for the service connection and hit Save
+
+## Creating the DevOps pipeline
+
+Click on Pipelines from the left-nav and then select "Create Pipeline".
+
+* From the "Where is your code?" screen, choose Github
+  * You may see a screen asking for authentication: "Authenticate to authorize access"
+* From the "Select a repository" screen, select the fork created above
+  * Select "Approve & Install Azure Pipelines" if required
+* From the "review your pipeline" screen, click the down-arrow next to Run and click "Save" - note that a number of variables need to be added before the first run
+
+### Update pipeline target condition
```

Comment:
We moved the target condition to an environment variable (line 106)..  so the entire "Update pipeline target condition" section goes away.

Reply:
Done. See commit [0212d04088f5955795e04005bb2a2cedb1bd989b](https://github.com/DirkSonguer/iot-edge-offline-dashboarding/commit/0212d04088f5955795e04005bb2a2cedb1bd989b).